public/css/UX.css

```diff
@@ -1,5 +1,4 @@
-.deleteOff {
-    color: red;
+.deleteOfflineProject {
```

Comment:
Maybe move to .scss files in `app/assets/stylesheets`

Reply:
that can be done when stylesheets of a simulator will be cleaned not a task for this pr I think